src/net/bpf.h

```diff
@@ -555,10 +560,10 @@ void	bpf_jit_freecode(bpfjit_func_t);
 
 #endif
 
-int	bpf_validate(const struct bpf_insn *, int);
-u_int	bpf_filter(const struct bpf_insn *, const u_char *, u_int, u_int);
+/* int	bpf_validate(const struct bpf_insn *, int); */
+/* u_int	bpf_filter(const struct bpf_insn *, const u_char *, u_int, u_int); */
```

Comment:
```suggestion
/* These are not used by bpfjit and would cause conflicts with pcap's copy of bpf.h */
/* int	bpf_validate(const struct bpf_insn *, int); */
/* u_int	bpf_filter(const struct bpf_insn *, const u_char *, u_int, u_int); */
```

Reply:
Ignore this. These lines are the results of the sed commands in update.sh.